pandas/tseries/offsets.py

```diff
@@ -929,8 +929,9 @@ def name(self):
         if self.isAnchored:
             return self.rule_code
         else:
+            month = liboffsets._int_to_month[self.n]
```

Comment:
prob should de-privatize these in offsets (``_int_to_month``)

Reply:
Sure